dashboard/src/components/tailwind/Input.js

```diff
@@ -0,0 +1,37 @@
+/*
+  This example requires some changes to your config:
+
+  ```
+  // tailwind.config.js
+  module.exports = {
+    // ...
+    plugins: [
+      // ...
+      require('@tailwindcss/forms'),
+    ],
+  }
+  ```
+*/
+// https://tailwindui.com/components/application-ui/forms/input-groups#component-2607d970262ada86428f063c72b1e7bd
+const Input = ({ id, name, label, placeholder, type, defaultValue, ...props }) => {
```

Comment:
```suggestion
const Input = ({ label, ...props }) => {
```
Du coup la seule propritété qui n'est pas dans props c'est celle là. Et toujours sur les composants, je me demande si on a besoin de mettre le label dedans si c'est pour pouvoir l'enlever.

En fait l'intérêt de ce composant est uniquement une liste de classes si on regarde, c'est la seule valeur ajoutée pour une nouvelle couche.

Et sous peu on voudra inventer une couche sur les classes aussi, et ça commencera à être bizarre.

C'est ouvert mais je me demande pourquoi on fait pas : 

```jsx
<input className={inputDefaultStyle} placeholder="x" />
```